src/main/scala/com/ing/wbaa/airlock/proxy/handler/FilterRecursiveListBucketHandler.scala

```diff
@@ -29,9 +36,9 @@ trait FilterRecursiveListBucketHandler extends LazyLogging {
    * @return for recursive request it returns filtered response for others the original response
    */
   protected[this] def filterResponse(request: HttpRequest, userSTS: User, s3request: S3Request, response: HttpResponse): HttpResponse = {
-    val isNoDelimiterParamAndIsReadAccess =
+    val isNoDelimiterParamAndIsReadAccessAndAccessTypeRead =
```

Comment:
This val is not very readable, maybe something shorter like notDelimiterParamWithRead?

Reply:
changed